CANTroller2/src/cantroller2.cpp

```diff
@@ -547,16 +539,13 @@ void loop() {
     adj = false;
     if (tuning_ctrl == EDIT && sim_edit_delta != 0) {  // Change tunable values when editing
         if (dataset_page == PG_RUN) {
-            if (selected_value == 4) adj_bool (&sim_joy, sim_edit_delta);
-            else if (selected_value == 5) adj_bool (&sim_pressure, sim_edit_delta);
-            else if (selected_value == 6) adj_bool (&sim_brkpos, sim_edit_delta);
-            else if (selected_value == 7) adj_bool (&sim_tach, sim_edit_delta);
-            else if (selected_value == 8) adj_bool (&sim_airflow, sim_edit_delta);
-            else if (selected_value == 9) adj_bool (&sim_speedo, sim_edit_delta);
-            else if (selected_value == 10) {
-                adj_val(&pot_overload, sim_edit_delta, 0, arraysize(sensorcard)-1);
-                if (pot_overload == pressure) pressure_sensor.set_source(ControllerMode::POT);
-            }
+            if (selected_value == 4) simulator.set_can_simulate(SimOption::joy, adj_bool(simulator.can_simulate(SimOption::joy), sim_edit_delta));
```

Comment:
Can we use a map eg 
```
const map = {
  4: SimOption::joy,
  5: SimOption::pressure
}
```
Or a switch with an enum or something? The numbers seem magic to me, and the code is repeated a lot here.

Reply:
we could stand a refactor of the touchscreen control logic for sure. the numbers are the indices of the touchscreen, but we could for sure map them to a better representation somewhere. I'm all for it, but I think that's also something we should tackle separately from this change.